pkgs/by-name/si/sile/package.nix

```diff
@@ -45,6 +45,7 @@ stdenv.mkDerivation (finalAttrs: {
   nativeBuildInputs = [
     zstd
     pkg-config
+    fontconfig # fc-match
```

Comment:
I tested locally, and reached the conclusion that `fontconfig` can (and therefor *should*?) be removed from `buildInputs`.

Reply:
Are you sure? The build result depends on fontconfig-lib.

Reply:
> Are you sure? The build result depends on fontconfig-lib.

You are right, I was mistaken. Thanks :+1: .